packages/sdk/src/evm/common/currency.ts

```diff
@@ -192,3 +192,22 @@ export async function normalizeAmount(
   const decimals = await contractWrapper.readContract.decimals();
   return utils.parseUnits(AmountSchema.parse(amount), decimals);
 }
+
+export function toEther(amount: Amount): string {
+  return utils.formatEther(AmountSchema.parse(amount));
+}
+
+export function toWei(amount: BigNumber | string): BigNumber {
```

Comment:
let's use `BigNumberish` here as the input type

Reply:
Ah, so sorry, I switched between them.
But I want to add `BigNumber | string)` in case the number format is something like this `1000000000000000000`. The `utils.formatEther` support both big numbers and string, so the user doesn't need to transfer string to a bigNumber. wdyt?

Reply:
@joaquim-verges I read my previous comments in bad English 😅 😅  So I updated it. Please tell me if it is unclear 🙏

Reply:
`BigNumberish` (not `BigNumber`) is a union of `string | number | BigNumber` which is what we use in a lot of places

Reply:
Ah okay got it, I will fix this👍